test/fixtures/docker-app/features/support/env.rb

```diff
@@ -0,0 +1,12 @@
+def output_logs
+  $docker_services.each do |service|
+    logged_service = service[:service] == :all ? '' : service[:service]
+    command = "logs -t #{logged_service}"
+    begin
+      response = run_docker_compose_command(service[:file], command)
+    rescue => exception
+      response = "Couldn't retreive logs for #{service[:file]}:#{logged_service}"
+    end
+    STDOUT.puts response.is_a?(String) ? response : response.to_a
+  end
+end
```

Comment:
\n

Reply:
This has been added, despite the lack of indication.